### PR TITLE
11 central db

### LINKED ISSUE #11: Central DB
Store dynamic data e.g. channels, slack members, current open views
Another docker container running mysql, sqllite or something similar
Ideally compatible with a python ORM for ease of interface development e.g. SQLAlchemy

### PR DESCRIPTION
**Pull request by samcorky**
Database is done - I am happy.

Implemented HTTP get space_state endpoint that returns whether the space is open or not.

### CODE REVIEW

**Review by sjefferson99 — COMMENTED** [review]
@sam57719 In general I think we need to ensure future PRs are smaller in scope. Ideally splitting into discrete features such as DB engine and driver, plugin updates etc.

I'll give it a once over and see what I can get from reading through and also give it a test on my local server. Would be good to get a review of the components added and design decisions on Wednesday so I'm up to speed ahead of merging in.

File: tests/mngo_contexct.py
Comment (sjefferson99) [inline tests/mngo_contexct.py:1]: @sam57719 Is this file name spelt correctly?

**Review by sjefferson99 — COMMENTED** [review]
@sam57719 Would be good to have some documentation updates to go with this PR. Probably just some lines in root dir readme detailing the features added, how to consume as a user and/or developer. You can make some headings up as you need like "plugins" or "API" or "tests" and I can merge my docs into them should I get to documenting the core code at some point.

**Comment by samcorky** [comment]
@sjefferson99 did some command line stuff on my pi 4 in France.

got it running - there seem to be a bunch of active connections still 

Had to pin the version number of the mongo container to the lastest one that wosk on a pi
see [this git comment](https://github.com/linuxserver/docker-unifi-network-application/issues/4#issuecomment-1719400553)

Ensured all containers on same docker network and told smib-slack what the hostname of the mongo-db container is

File: tests/mngo_contexct.py
Comment (samcorky) [reply tests/mngo_contexct.py:1]: Nope 😂

Comment (sjefferson99) [reply tests/mngo_contexct.py:1]: Issue raised to clear out test folder #52

**Review by sjefferson99 — APPROVED** [review]
Happy with PR. Needs to be smaller in future and random tests can go in random branches.

**Pull request merged**